docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,106 @@
+
+- **SSO**: Properly setting up Grafana with your current IAM solution enables users to access Grafana with the same credentials they use for other applications.
+
+- **Scalability**: Adding, updating, or removing users from your user database is immediately reflected in Grafana.
+
+In order to plan an integration with Grafana, assess your organization's current needs, requirements and any existing IAM solutions being used. This should include which set of roles and permissions will be match to each type of user and which groups of users share the same set of permissions and access to shared resources.
+
+After considering needs, chosen IAM solution and taking into consideration the security requirements, the last step is to test it out thoroughly before deploying it to a production environment.
+
+## Internal vs external users
+
+As a first step, determine where your Grafana users are located. Are the users located within your organization, or are they outside your organization?
+
+If the users are within your organization, Grafana might be able to integrate with those users through an identify provider.
+
+If the users are outside your organization, you must provide anonymous access to Grafana, which is not enabled by default.
+
+For information about enabling anonymous access, refer to the [documentation](../../setup-grafana/configure-security/configure-authentication/_index.md#anonymous-authentication)
+
+## Ways to organize users
+
+Organize users in subgroups that are sensible to the organization. These are some examples:
+
+- **Security**: Different groups of users or customers should only have access to their intended resources.
+- **Simplicity**: Reduce the scope of dashboards and resources available.
+- **Cost attribution**: Track and bill costs to their customers, departments, or divisions.
+- **Customization**: Each group of users could have a personalized experience like different dashboards or theme colours.
+
+### Users in Grafana Teams
+
+You can organize users into [Teams] and assign them roles and permissions reflecting the current organization. For example, instead of assigning five users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team. A user can belong to multiple teams.
+
+A user can be a Member or an Administrator for a given team. Members of a team inherit permissions from the team, but they cannot edit the team itself. Team Administrators can add members to a team and update its settings, such as the team name, team members' team roles, UI preferences, and the default dashboard to be displayed upon login for the team members.
+
+Teams are a perfect solution for working with a small subset of users. Teams can share resources with other teams.
+
+### Users in Grafana Organizations
```

Comment:
Let's chat about orgs, there are some things I would reword slightly:
* orgs don't really isolate users from resources, but rather isolate the resources themselves;
* we could reword the reasoning for suggesting teams over orgs, as it's not just scalability.